content/docs/manuals/worker-configuration.md

```diff
@@ -0,0 +1,198 @@
+```
+
+and then re-running the above install command.
+{{< /callout >}}
+
+## Configuration
+
+The NodeEngine and NetManager are implemented as systemd services. While it is possible to run the former without the latter, this configuration is impractical for most applications, therefore the NodeEngine service internally calls on the NetManager service. The NetManager cannot be run without the NodeEngine.
+
+### Commands supported by both components
+
+| Command    | Description                                                    |
+|------------|----------------------------------------------------------------|
+| `help`     | Displays an overview of the service and the available commands |
+| `version`  | Prints the current version                                     |
+| `status`   | Shows the systemd service status                               |
+| `logs`     | Tails the service logs                                         |
+
+{{< callout context="note" title="Elevated Privileges" icon="outline/info-circle" >}}
+The NodeEngine and NetManager require elevated privileges for many operations. Therefore it is often necessary to execute commands as the root user with `sudo`.
+{{< /callout >}}
+
+### Configuring the NetManager
+
+You can configure the NetManager by editing the file at `/etc/netmanager/netcfg.json`. Here you can configure the following properties:
+
+| Property          | Description                                                                                                        |
+|-------------------|--------------------------------------------------------------------------------------------------------------------|
+| NodePublicAddress | The address by which the worker node can be reached                                                                |
+| NodePublicPort    | The port by which the worker node can be reached (default 50103)                                                   |
+| ClusterUrl        | The URL/address by which the cluster can be reached                                                                |
+| ClusterMqttPort   | The port by which the cluster MQTT broker can be reached (default 10000)                                           |
+| DefaultInterface  | Should the system have multiple default interfaces (e.g. `eth0` and `wlan0`) the appropriate one must be specified |
+| Debug             | Toggles more verbose logging                                                                                       |
+| MqttCert          | The path to the certificate file to facilitate [MQTTS](../networking-internals/mqtt-authentication)                |
+| MqttKey           | The path to the key file to facilitate [MQTTS](../networking-internals/mqtt-authentication)                        |
+
+### Running the NodeEngine
+
+Once the worker components have been installed the NodeEngine can be run with `sudo NodeEngine`. This will start the NodeEngine service per the default configuration.
```

Comment:
Frankly speaking, I would avoid duplicating any information that was already mentioned here https://www.oakestra.io/docs/getting-started/oak-environment/add-edge-devices-workers-to-your-setup/#add-a-worker-node-to-your-cluster

Instead you can easily create a callout informing readers that to first get familiar with the basics.
Similar to what I did with the CLI https://www.oakestra.io/docs/manuals/cli/cli-overview/